package/post-install.sh

```diff
@@ -66,9 +70,13 @@ function install_conf_d_files() {
     println "\n"
 
     # Detect installed SAPI's
-    SAPI_CONFIG_DIRS=(${PHP_CFG_DIR})
-    SAPI_DIR=${PHP_CFG_DIR%cli/conf.d}
+    SAPI_DIR=${PHP_CFG_DIR%/*/conf.d}/
+    SAPI_CONFIG_DIRS=()
     if [[ "$PHP_CFG_DIR" != "$SAPI_DIR" ]]; then
+        # Detect CLI
+        if [[ -d "${SAPI_DIR}cli/conf.d" ]]; then
```

Comment:
Great question! Before this PR, the post-install hook was essentially targeting `$(which php)` which typically exposes INI files that are specific to just the CLI SAPI. Since FPM often has its own separate INI files (like the `ppa:ondrej/php` repo) targeting only the CLI would not properly install ddtrace for FPM. So that's why the post-install hook probes for other SAPI's of the same version.

This PR ensures the probing functionality can also do the reverse, so that if the FPM binary is targeted, the CLI of the same PHP version will also get included during the install.

There are other common organizational structures of INI files that the post-install hook does not accommodate for, but this at least gives us a good starting point to start probing for more configurations that we can test. :)